packages/opentelemetry-core/src/platform/node/ShutdownNotifier.ts

```diff
@@ -14,13 +14,22 @@
  * limitations under the License.
  */
 
+let shutDownListeners: Array<() => void> = [];
+process.once('SIGTERM', function () {
```

Comment:
I think it should be here `on` not `once`

Reply:
Think you're right